hyp3_rtc_gamma/__main__.py

```diff
@@ -134,251 +124,82 @@ def main_v2():
-            cfg['out_path'] = out_path
-
-            with get_db_connection('hyp3-db') as conn:
-                clip_tiffs_to_roi(cfg, conn, product_dir)
-
-                log.debug('Renaming ' + product_dir + ' to ' + out_path)
-                os.rename(product_dir, out_path)
-
-                browse_path = find_png(out_path)
-                cfg['attachment'] = browse_path
-                add_browse(cfg, 'LOW-RES', browse_path)
-
-                find_browses(cfg, out_path)
-                add_citation(cfg, out_path)
-                zip_dir(out_path, zip_file)
-
-                cfg['final_product_size'] = [os.stat(zip_file).st_size, ]
-                cfg['original_product_size'] = 0
-
-                record_metrics(cfg, conn)
-                if 'lag' in cfg and 'email_text' in cfg:
-                    cfg['email_text'] += "\n" + "You are receiving this product {0} after it was acquired.".format(
-                        cfg['lag'])
-
-                upload_product(zip_file, cfg, conn, browse_path=browse_path)
-                success(conn, cfg)
+        logging.basicConfig(format='%(asctime)s - %(levelname)s - %(message)s',
+                            datefmt='%m/%d/%Y %I:%M:%S %p', level=logging.INFO)
+        logging.info(f'Processing GAMMA RTC "{cfg["sub_name"]}" for "{cfg["username"]}"')
+
+        granule = cfg['granule']
+        if not re.match('S1[AB]_.._[SLC|GRD]', granule):
+            raise ValueError(f'Invalid granule, only S1 SLC and GRD data are supported: {granule}')
+
+        res = get_extra_arg(cfg, 'resolution', '30m')
+        if res not in ('10m', '30m'):
+            raise ValueError(f'Invalid resolution, valid options are 10m or 30m: {res}')
+
+        granule_url = get_download_url(granule)
+        granule_zip_file = download_file(granule_url, chunk_size=5242880)
+
+        args = {
+            'in_file': granule_zip_file,
+            'res': int(res.rstrip('m')),
+            'match_flag': extra_arg_is(cfg, 'matching', 'yes'),
+            'pwr_flag': extra_arg_is(cfg, 'power', 'yes'),
+            'gamma_flag': extra_arg_is(cfg, 'gamma0', 'yes'),
+            'lo_flag': res == '30m',
+            'filter_flag': extra_arg_is(cfg, 'filter', 'yes'),
+        }
+        product_dir, product_name = rtc_sentinel_gamma(**args)
+
+        logging.info(f'Renaming {product_dir} to {product_name}')
+        os.rename(product_dir, product_name)
+        product_dir = product_name
+
+        if extra_arg_is(cfg, 'include_dem', 'no'):
+            find_and_remove(product_dir, '*_dem.tif*')
+        if extra_arg_is(cfg, 'include_inc', 'no'):
+            find_and_remove(product_dir, '*_inc_map.tif*')
+
+        zip_file = make_archive(base_name=product_dir, format='zip', base_dir=product_dir)
+        cfg['final_product_size'] = [os.stat(zip_file).st_size, ]
+        cfg['attachment'] = find_png(product_dir)
+
+        for handler in log.handlers:
+            log.removeHandler(handler)
```

Comment:
Should this be up where you setup the logging? 
https://github.com/ASFHyP3/hyp3-rtc-gamma/pull/141/files#diff-8be91576c32694692cf90f1752b8c91fR150-R152